plugwise/smile.py

```diff
@@ -592,40 +612,30 @@ async def set_switch_state(self, appl_id, members, model, state):
         if members is not None:
             return await self._set_groupswitch_member_state(members, state, switch)
 
-        locator = f'appliance[@id="{appl_id}"]/{switch.actuator}/{switch.func_type}'
-        switch_id = self._appliances.find(locator).attrib["id"]
-        uri = f"{APPLIANCES};id={appl_id}/{switch.device};id={switch_id}"
+        locator: str = (
+            f'appliance[@id="{appl_id}"]/{switch.actuator}/{switch.func_type}'
+        )
+        switch_id: str = self._appliances.find(locator).attrib["id"]
+        uri: str = f"{APPLIANCES};id={appl_id}/{switch.device};id={switch_id}"
         if self._stretch_v2:
             uri = f"{APPLIANCES};id={appl_id}/{switch.device}"
-        data = f"<{switch.func_type}><{switch.func}>{state}</{switch.func}></{switch.func_type}>"
+        data: str = f"<{switch.func_type}><{switch.func}>{state}</{switch.func}></{switch.func_type}>"
 
         if model == "relay":
             locator = (
                 f'appliance[@id="{appl_id}"]/{switch.actuator}/{switch.func_type}/lock'
             )
-            lock_state = self._appliances.find(locator).text
+            lock_state: str = self._appliances.find(locator).text
             # Don't bother switching a relay when the corresponding lock-state is true
             if lock_state == "true":
                 return False
 
         await self._request(uri, method="put", data=data)
         return True
 
-    async def _set_preset_legacy(self, preset):
-        """Set the given Preset on the relevant Thermostat - from DOMAIN_OBJECTS."""
-        locator = f'rule/directives/when/then[@icon="{preset}"].../.../...'
-        if (rule := self._domain_objects.find(locator)) is None:
-            return False
-
-        uri = f"{RULES}"
-        data = f'<rules><rule id="{rule.attrib["id"]}"><active>true</active></rule></rules>'
-
-        await self._request(uri, method="put", data=data)
-        return True
-
-    async def delete_notification(self):
+    async def delete_notification(self) -> bool:
         """Delete the active Plugwise Notification."""
-        uri = f"{NOTIFICATIONS}"
+        uri: str = f"{NOTIFICATIONS}"
```

Comment:
The type can be inferred here, additionally, there is no need to interpolate this string.
```suggestion
        uri = NOTIFICATIONS
```